E01-Geometry/Benas/src/shapes/Rhombus.php

```
@@ -0,0 +1,29 @@
+<?php
+
+
+class Rhombus extends Shape
+{
+    public float $edge;
+    public float $degree=33;
+
+    public function __construct(float $edge)
+    {
+        if($edge>0) {
```

Same here. Fail early